plugins/speculation-rules/readme.txt

```diff
@@ -9,15 +9,15 @@ License:           GPLv2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
 Tags:              performance, javascript, speculation rules, prerender, prefetch
 
-Uses the Speculation Rules API to prerender linked URLs upon hover by default.
+Enables browsers to speculatively prerender or prefetch pages when hovering over links.
 
 == Description ==
```

Comment:
Ditto above about mentioning the old name.

Reply:
Done in https://github.com/WordPress/performance/pull/1101/commits/0c04a92f8cdf1eb4067a0ca11f2c0b92ee4a7b24